src/roadmap/config.py

```diff
@@ -24,4 +26,19 @@ def database_url(self) -> PostgresDsn:
         )
 
 
-SETTINGS = Settings()
+_overrides = {}
```

Comment:
It seems a bit messy having this as a module level global. How about adding a factory method to the `Settings` class that handles this?
```python
class Settings(BaseSettings):
    ...
    @classmethod
    def create(cls):
        if isClowderEnabled():
            db = LoadedConfig.database
            return cls(
                db_name=db.name,
                db_user=db.username,
                db_password=SecretStr(db.password),
                db_host=db.hostname,
                db_port=db.port,
            )

        return cls()
```